firebase-config/src/test/java/com/google/firebase/remoteconfig/internal/ConfigFetchHttpClientTest.java

```diff
@@ -240,22 +242,22 @@ public void fetch_localeUsesToStringBelowLollipop() throws Exception {
   }
 
   @Test
-  public void fetch_instanceIdIsNull_throwsFRCClientException() throws Exception {
+  public void fetch_installationIdIsNull_throwsFRCClientException() throws Exception {
     setServerResponseTo(noChangeResponseBody, SECOND_ETAG);
 
     FirebaseRemoteConfigClientException frcException =
-        assertThrows(FirebaseRemoteConfigClientException.class, () -> fetchWithoutIid());
+        assertThrows(FirebaseRemoteConfigClientException.class, () -> fetchWithoutInstallationId());
 
-    assertThat(frcException).hasMessageThat().contains("instance id is null");
+    assertThat(frcException).hasMessageThat().contains("installation id is null");
   }
 
   @Test
-  public void fetch_instanceIdTokenIsNull_doesNotThrowException() throws Exception {
+  public void fetch_installationAuthTokenIsNull_doesNotThrowException() throws Exception {
     setServerResponseTo(noChangeResponseBody, SECOND_ETAG);
 
-    FetchResponse fetchResponse = fetchWithoutIidToken();
+    FetchResponse fetchResponse = fetchWithoutInstallationAuthToken();
 
-    assertWithMessage("Fetch() failed with null instance id token!")
+    assertWithMessage("Fetch() failed with null installation auth token!")
```

Comment:
Thinking (no action required): this is public error message, so the phrasing should match the public docs. The [FirebaseInstallations.getToken docs](https://firebase.google.com/docs/reference/android/com/google/firebase/installations/FirebaseInstallations#getToken(boolean)) describe "authentication token for the Firebase installation", which matches the phrasing here.

Reply:
Great callout, thank you!
We had a meeting recently with DevRel and Tech Writers and decided to name the authentication tokens for Firebase installations: "installation auth tokens".
So, the action item is probably on us in the FIS team.